src/util/Enums.js

```diff
@@ -78,4 +79,12 @@ export default {
   CUSTOM_URI_CHALLENGE: 'CUSTOM_URI',
   UNIVERSAL_LINK_CHALLENGE: 'UNIVERSAL_LINK',
   APP_LINK_CHALLENGE: 'APP_LINK',
+
+  // Initial Flows
+  DEFAULT_FLOW: 'default',
+  PROCEED_WITH_FLOW: 'proceed',
+  LOGIN_FLOW: 'login',
+  REGISTRATION_FLOW: 'signup',
+  RESET_PASSWORD_FLOW: 'resetPassword',
+  UNLOCK_ACCOUNT_FLOW: 'unlockAccount',
```

Comment:
I would create an object for these flows and export it..

CUSTOM_CONFIGURED_FLOWS = {
  ... : ...
}


You can add this to `RemediationConstants.js` instead of `Enum.js` ?

Reply:
My hesitation for doing so is `RemediationConstants.js` is in the `ion` directory. The `flow` configuration and it's logic all happens _before_ the `ion` transformation for `idx` responses. These enum values are just possible configuration values for `flow`

Reply:
I added another comment, which aims to explain that

Reply:
worth noting here that flow is a *client-side only* concept, it is not part of server remediations or IDX spec. A major purpose of this `flow` concept is to prevent a transaction in-progress from being continued when the user intention is to start a new transaction. (such as switching from login to register)

Reply:
Got it, than it make sense not to put this under remediation constants.. 

But let's make this an object, because i foresee there would be many keys in future..

```
CONFIGURED_FLOW = {
  DEFAULT : 'default'
  PROCEED: 'proceed',
  LOGIN: 'login',
  REGISTRATION: 'signup',
  RESET_PASSWORD: 'resetPassword',
}

and it could be used as..

CONFIGURED_FLOW.PROCEED

we can avoid adding `_FLOW` in every key
```

Reply:
Also lets not add them to Enums.js, instead put them in `Constants.js`... Enums.js is not under `v2`

Reply:
done

Reply:
there is one `Constant.js` already exists under `v2/view-builder/utils` see if you want to use it.. your call.

Reply:
I think it makes sense to have a separate `constants.js` file for this. It will make the eventual `auth-js` transition slightly easier